crates/storage_models/src/enums.rs

```diff
@@ -3,7 +3,7 @@ pub mod diesel_exports {
     pub use super::{
         DbAttemptStatus as AttemptStatus, DbAuthenticationType as AuthenticationType,
         DbCaptureMethod as CaptureMethod, DbConnectorType as ConnectorType,
-        DbCountryCode as CountryCode, DbCurrency as Currency, DbDisputeStage as DisputeStage,
+        DbCountryAlpha2 as CountryAlpha2, DbCurrency as Currency, DbDisputeStage as DisputeStage,
```

Comment:
As far as I understand, this will cause the `schema.rs` file to go to an inconsistent state (and possibly fail to build) when you run `diesel migration run` on a fresh clone of the repository, since the database type would be named `CountryCode`, while our code expects it to be `CountryAlpha2`.

There are a few ways to handle this, I feel the simplest way is to just rename the database type to be also called `CountryAlpha2`.

Reply:
So `DbCountryCode as CountryAlpha2` would have been better if we're not renaming in the database?  
Also for some reason, on my local, it shows 60+ problems with this branch while it passes all the test cases here. `cargo clean` seems to fix the issue locally btw.

Reply:
Yes you can add a migration for this. It's always better to have proper naming and be explicit rather than having the developer make an effort to look up the database type to know its details.